### PR TITLE
feat: Support local vscode using remote ssh mode to session container

### PR DESCRIPTION
**Pull request by studioego**
Support local vscode using remote ssh mode to session container

 

- [x] agent: Generate a human redable  password in entrypoint.sh and store it as .password in the home directory (just like id_container) when the session container starts.

- [x] agent: enable password logins on dropbear SSH
 
- [x] manager: Add .password as one of the prohibited names for vfolders.

After closed the Backend.AI web ui PR https://github.com/lablup/backend.ai-webui/pull/1487. then the issue 
 https://github.com/lablup/backend.ai/issues/646 could be closed.

### CODE REVIEW

File: src/ai/backend/agent/agent.py [hunk 1]
```diff
@@ -420,13 +420,20 @@ def find_artifacts(pattern: str) -> Mapping[str, str]:
         )
         entrypoint_sh_path = self.resolve_krunner_filepath("runner/entrypoint.sh")
 
+        pantompass_path = self.resolve_krunner_filepath("runner/fantompass.py")
```
Comment (kyujin-cho) [inline src/ai/backend/agent/agent.py]: Looks like typos are here...

```suggestion
        fantompass_path = self.resolve_krunner_filepath("runner/fantompass.py")
```

File: src/ai/backend/agent/agent.py [hunk 2]
```diff
@@ -420,13 +420,20 @@ def find_artifacts(pattern: str) -> Mapping[str, str]:
...
+        words_json_path = self.resolve_krunner_filepath("runner/words.json")
+
         if matched_libc_style == "musl":
             terminfo_path = self.resolve_krunner_filepath("runner/terminfo.alpine3.8")
             _mount(MountTypes.BIND, terminfo_path, "/home/work/.terminfo")
 
         _mount(MountTypes.BIND, dotfile_extractor_path, "/opt/kernel/extract_dotfiles.py")
         _mount(MountTypes.BIND, entrypoint_sh_path, "/opt/kernel/entrypoint.sh")
         _mount(MountTypes.BIND, suexec_path, "/opt/kernel/su-exec")
+        _mount(MountTypes.BIND, pantompass_path, "/opt/kernel/fantompass.py")
```
Comment (kyujin-cho) [inline src/ai/backend/agent/agent.py]: Same here.
```suggestion
        _mount(MountTypes.BIND, fantompass_path, "/opt/kernel/fantompass.py")
```

**Review by kyujin-cho — COMMENTED** [review]
Just a small typo!

File: src/ai/backend/agent/agent.py (hunk 1 above)
Comment (studioego) [reply src/ai/backend/agent/agent.py]: fix typos

**Comment by kyujin-cho** [comment]
ref: lablup/backend.ai-webui#1487

**Pull request merged**